services/QuillLMS/app/services/demo/report_demo_creator.rb

```diff
@@ -378,15 +378,14 @@ def self.create_replayed_activity_session(student, classroom_unit)
     replayed_session = ActivitySession.unscoped.where({activity_id: REPLAYED_ACTIVITY_ID, user_id: REPLAYED_SAMPLE_USER_ID, is_final_score: true}).first
     student_id = student.id
     act_session = ActivitySession.create({activity_id: REPLAYED_ACTIVITY_ID, classroom_unit_id: classroom_unit.id, user_id: student.id, state: "finished", percentage: replayed_session&.percentage})
-    replayed_session&.old_concept_results&.each do |cr|
+    replayed_session&.concept_results&.each do |cr|
       values = {
         activity_session_id: act_session.id,
         concept_id: cr.concept_id,
-        metadata: cr.metadata,
-        question_type: cr.question_type
+        metadata: cr.legacy_format[:metadata],
+        question_type: cr.concept_result_question_type&.text
```

Comment:
Is this expected that `question_type` uses `&.text` whereas the report_demo_ap_creator and create_admin_report do not.

Reply:
Good catch on this and the next comment.  Looks like there's no spec coverage on the additional files, so they didn't get caught by automatic testing.  I'll make sure to do manual testing around it on staging.